tor/strings/en_US.yml

```diff
@@ -259,6 +259,19 @@ formatting_issues:
 
     \    int x = 0;\n
     \   int y = x * 100;"
+  unescaped_header: "**Unescaped header syntax**: It looks like you have an escaped hashtag in your transcription, this may cause it to render as a header instead of a raw hashtag. For example:
+
+
+    \    #Text
+
+
+    would render as:
+
+
+    #Text
+
+
+    To fix this, add a backslash in front of the \\# like so: `\\#Text`. If you meant for it to render as a header, please add a space after the \\# to avoid this warning. i.e. `# Text`."
```

Comment:
The terminoligy is inconsistent here, the function is named `unescaped_hashtag` and the here it's `unescaped_header`.

I think the text might also be a bit confusing: It says "you have an escaped hashtag", but instead it *needs* to be escaped and is not at the moment.
I would also use the term "heading" instead of "header", to avoid confusion with the header at the top of the transcription.